pluginLauncher/lib/source/DobbyRdkPluginManager.cpp

```diff
@@ -497,6 +497,20 @@ std::shared_ptr<IDobbyRdkLoggingPlugin> DobbyRdkPluginManager::getContainerLogge
     return containerLogger;
 }
 
+// -----------------------------------------------------------------------------
+/**
+ *  @brief Set the exit status of the container
+ *
+ *  @param[in]  status        The exit status of the container.
+ *
+ *  @return void
+ */
+void DobbyRdkPluginManager::setExitStatus(int status) const
+{
+    AI_LOG_INFO("STATUS=%d",status);
```

Comment:
Isn't this the same message as in `DobbyManager::handleContainerTerminate`? As there you can get more usefull date i.e. name of the container, you can probably get rid of this message here, as it is duplicate.